extension/content/enginesView.css

```diff
@@ -2,38 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#locale-region-settings {
-  display: grid;
-  justify-content: space-between;
-  align-items: center;
-  width: max-content;
-  margin: 0 auto;
-  grid-template-rows: repeat(2, auto);
-  grid-template-columns: repeat(4, calc(25% - 6em / 4));
-  column-gap: 2ch;
-  width: 80%;
-}
-
-#application-name-wrapper,
-#region-select-wrapper,
-#locale-select-wrapper {
-  grid-row: 1 / 3;
+#locale-region-settings > table {
```

Comment:
As we're adding more things, this section now contains different types of information beyond just locale and region, including application details, update channels, distribution IDs, experiment IDs, and versions. The current ID #locale-region-settings feels a bit narrow in scope.

How about renaming it to `#environment-settings` or something similar that sounds more general and broader in scope? 

What do you think?